localisation/00_wc_sad.csv

```diff
@@ -749,7 +749,21 @@ RULE_STATIC_PORTRAITS_ENABLED;Enabled;;Aktiviert;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_ENABLED_DESC;Static portraits for certain lore characters are �Genabled�!.;;Statische Portraits f�r bestimmte historische Charaktere sind �Gaktiviert�!.;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_DISABLED;Disabled;;Deaktiviert;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_DISABLED_DESC;Static portraits for certain lore characters are �Rdisabled�!.;;Statische Portraits f�r bestimmte historische Charaktere sind �Rdeaktiviert�!.;;;;;;;;;;;x
-;;;;;;;;;;;;;;
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
+writing_whispers;Hears Whispers;;;;;;;;;;;;;x
+writing_whispers_desc;This character hears whispers of the Old Gods.;;;;;;;;;;;;;x
+whispers_tooltip_tip;Quality of the book depends on amount of your �YTrueReligion.GetPietyName�!.;;;;;;;;;;;;;x
+EVTDESC_WCCSO.10200;The whispers of your masters have been getting stronger recently. They are pouring their eldritch knowledge into your mind but it's simply too much for your mind to take and you're unable to make sense of any of it. You need to focus on one portion of them and write them down, but first you need a writing material.;;;;;;;;;;;;;x
+EVTOPTA_WCCSO.10200;That one servant girl had such a smooth skin.;;;;;;;;;;;;;x
+EVTDESC_WCCSO.10201;After spending some time, trying to filter out one of the voices and gathering all needed materials, you can now write down its whispers.;;;;;;;;;;;;;x
+EVTOPTA_WCCSO.10201;It speaks of wars from long-forgotten eons.;;;;;;;;;;;;;x
+EVTOPTB_WCCSO.10201;It speaks of temple-cities made of black obsidian.;;;;;;;;;;;;;x
+EVTOPTC_WCCSO.10201;It speaks of a creature with thousand forms.;;;;;;;;;;;;;x
+EVTOPTD_WCCSO.10201;It speaks of forgotten starry secrets.;;;;;;;;;;;;;x
+EVTDESC_WCCSO.10202_martial;As time progresses, the whispers get stronger and turn into visions and before long you can see countless legions of the Void and terrible monstrosities designed to wage never-ending war in the name of their masters.;;;;;;;;;;;;;x
+EVTDESC_WCCSO.10202_stewardship;As time progresses, the whispers get stronger and turn into visions and before long you can see black temple-cities swarming with pyramids of incomprehensible geometry and obelisks portraying acts of eldritch depravity.;;;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCCSO.10202_stewardship;As time progresses, the whispers get stronger and turn into visions. Before long you can see black temple-cities filled with pyramids of incomprehensible geometry and obelisks portraying acts of eldritch depravity.;;;;;;;;;;;;;x
```

Reply:
i like swarming cause it sounds more disgusting but if it really doesnt make sense i can change it

Reply:
I mean, it makes sense in that I understand what you are saying. Having said that, swarming implies movement. Overrun maybe, instead of filled? Overflowing could also work. It's not a major deal, in that it technically works, it is just awkward. If you really like it the way it is, I doubt anyone would have problems with it. Just felt I should point it out.

Reply:
Overflowing sounds nice here 👍

Reply:
how about crowded?

Reply:
sounds to me like "less" then overflowing/swarming etc, but I'm ok with that, too

Reply:
Crowded could work too, yeah.